pkg/test/resourcefixture/testdata/basic/accesscontextmanager/v1beta1/accesscontextmanagerserviceperimeterresource/create.yaml

```diff
@@ -12,11 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: accesscontextmanager.cnrm.cloud.google.com/v1alpha1
+apiVersion: accesscontextmanager.cnrm.cloud.google.com/v1beta1
 kind: AccessContextManagerServicePerimeterResource
 metadata:
   name: accesscontextmanagerserviceperimeterresource-${uniqueId}
 spec:
   perimeterNameRef:
-    name: accesscontextmanagerserviceperimeter-${uniqueId}
-  resource: projects/${projectNumber}
+    # Referencing a resource name leads to recursive reference and KCC does not support the feature for now.
```

Comment:
Could you create a GH issue to track this feature gap and then add a TODO comment here (so you can leverage the GH issue to describe the detailed issue and keep the comment simple).

Reply:
Yeah, created https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/1309 and added TODO